Python/ceval.c

```diff
@@ -506,7 +522,14 @@ Py_MakePendingCalls(void)
     if (busy)
         return 0;
     busy = 1;
+
     UNSIGNAL_PENDING_CALLS();
+    if (PyErr_CheckSignals() < 0) {
```

Comment:
Please copy the comment before the if:

+    /* Python signal handler doesn't really queue a callback: it only signals
+       that a signal was received, see _PyEval_SignalReceived(). */